src/test/groovy/netflix/nebula/NebulaKotlinNodepPluginIntegrationSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package netflix.nebula
+
+
+import nebula.test.IntegrationSpec
+
+class NebulaKotlinNodepPluginIntegrationSpec extends IntegrationSpec {
+    String kotlinVersion
+
+    def setup() {
+        kotlinVersion = NebulaKotlinPlugin.loadKotlinVersion()
+        buildFile << """\
+        apply plugin: 'nebula.kotlin-nodep'
+
+        repositories {
+            mavenCentral()
+            maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+        }
+        """.stripIndent()
+    }
+
+    def 'plugin applies'() {
+        given:
+        buildFile.delete()
+        buildFile << """
+        apply plugin: 'nebula.kotlin-nodep'
+        """
+
+        when:
+        runTasksSuccessfully('help')
+
+        then:
+        noExceptionThrown()
+    }
+
+    def 'default standard library is not added'() {
+        given:
+        buildFile << """
+        sourceCompatibility = JavaVersion.VERSION_1_6
+        """
+
+        when:
+        def resultCompileClasspath = runTasksSuccessfully('dependencies', '--configuration', 'compileClasspath')
+
+        then:
+        !resultCompileClasspath.standardOutput.contains("\\--- org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion\n")
+    }
+
+    def 'jdk7 standard library is not added when source compatibility is VERSION_1_7'() {
+        given:
+        buildFile << """
+        sourceCompatibility = JavaVersion.VERSION_1_7
+        """
+
+        when:
+        def resultCompileClasspath = runTasksSuccessfully('dependencies', '--configuration', 'compileClasspath')
+
+        then:
+        !resultCompileClasspath.standardOutput.contains("\\--- org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion\n")
+    }
+
+    def 'jdk8 standard library is not added when source compatibility is VERSION_1_8 for test configuration only'() {
+        given:
+        buildFile << """
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        """
+
+        when:
+        def resultCompileClasspath = runTasksSuccessfully('dependencies', '--configuration', 'compileClasspath')
+
+        then:
+        !resultCompileClasspath.standardOutput.contains("\\--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion\n")
+    }
```

Comment:
Can you add a test case in here where ppl apply the no-dep plugin and add dependencies to a configuration explicitly?

Reply:
added here https://github.com/nebula-plugins/nebula-kotlin-plugin/pull/34/commits/4dc0c5f44bf91751f1c33a462bc05632c60fbce4

Reply:
🎉